packages/stores/src/account/base.ts

```diff
@@ -1272,6 +1272,16 @@ export class AccountStore<Injects extends Record<string, any>[] = []> {
           })
         : undefined;
 
+      apiClient("/api/transaction-scan", {
```

Comment:
Is this just logging? Why not await? Is it validating the tx, and returning an error HTTP status if invalid?

Reply:
It's just to send data to Blockaid so they can train their model, likely soon well need to do something with the data